### PR TITLE
Change `gets()` to `fgets()` in user prompts

### PR DESCRIPTION
**Pull request by dey4ss**

### CODE REVIEW

**Review by Bouncner — APPROVED** [review]

File: makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ WORKLOAD =SSBM
 # add -EDTERABYTE if orderkey will execeed 32 bits (SF >= 300)
 # and make the appropriate change in gen_schema() of runit.sh
 # HYRISE: Change -O to -O3. Add -Wno-dangling-else, -Wno-int-to-void-pointer-cast, and -Wno-void-pointer-to-int-cast because they are all over the codebase.
-CFLAGS	= -O3 -DDBNAME=\"dss\" -D$(MACHINE) -D$(DATABASE) -D$(WORKLOAD) -Wno-dangling-else -Wno-int-to-void-pointer-cast -Wno-void-pointer-to-int-cast
+CFLAGS	= -O3 -DDBNAME=\"dss\" -D$(MACHINE) -D$(DATABASE) -D$(WORKLOAD) -Wno-dangling-else -Wno-int-to-void-pointer-cast -Wno-void-pointer-to-int-cast -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast
```
Comment (Bouncner) [inline makefile:19]: Actually, why is `-O3` not also an optional flag just like the compiler? We would keep everything as is but still allow "better" builds for Hyrise.

(not that is matters ... won't block this PR).

Comment (dey4ss) [reply makefile:19]: Thought about the same. However, we only call the SSB and JCC-H dbgens via CLI. No way we ever step into the code while debugging, so I see no advantage of not always using `-O3`.

Comment (Bouncner) [reply makefile:19]: Yes, for us it does not matter. I was rather thinking about other people potentially using this fork.

Comment (dey4ss) [reply makefile:19]: Do you think they want to debug this? I mean, we can add a switch for either setting `-g` or `-O3`, but I guess anyone who wants to step into this ~abomination~ beauty can set compiler flags on their own.

Comment (Bouncner) [reply makefile:19]: Hmm ... okay. When somebody really wants to debug, then she will need to dig into this part anyways. Good point.

**Pull request merged**